### PR TITLE
Removed files and applied enum pattern

### PR DESCRIPTION
**Pull request by jchen006**
@snallapa basically removed some unneeded files and switch to enums

### CODE REVIEW

File: app/static/js/components/Experiments/Experiments.jsx [hunk 1]
```diff
@@ -4,6 +4,7 @@ import { withStyles } from "@material-ui/core/styles";
 import IngredientFrequency from "./IngredientFrequency/IngredientFrequencyBarChart";
 import WordCloud from "./WordCloud/WordCloud";
 import styles from "./styles";
+import routes from '../enums/experimentRoutes.js'
```
Comment (snallapa) [inline app/static/js/components/Experiments/Experiments.jsx]: webpack resolve the folder components so you can do components/enums/experimentRoutes if you want. I tend to like that more than using ..

Comment (jchen006) [reply app/static/js/components/Experiments/Experiments.jsx]: Should we add more of these webpack aliases or would it get too wild at some point?

**Comment by jchen006** [comment]
@snallapa merge?

File: app/static/js/components/Experiments/Experiments.jsx (hunk 1 above)
Comment (snallapa) [reply app/static/js/components/Experiments/Experiments.jsx]: this one is probably enough

**Pull request merged**